testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -463,167 +421,172 @@ def test_coordinates(self):
                 ts_orig.frame))
 
 
-class TestDCDCorrel(TestCase):
```

Comment:
Ok sensible.

(I think I made exactly the same comment 2 days ago but I replied by email and it apparently never made it into the GitHub comments.)